### PR TITLE
Fix English (en) pronouns

### PR DESCRIPTION
**Pull request by lyczak**
Replaced gendered (he/him/his) pronouns in `ttt.phrases.txt` with gender-neutral (they/them/theirs) pronouns. Also fixed small pronoun-typo in `ttt_rdm.phrases.txt`.

### CODE REVIEW

File: addons/sourcemod/translations/ttt.phrases.txt [hunk 1]
```diff
@@ -239,7 +239,7 @@
 
 	"The weapon used has been: himself (suicide)"
```
Comment (c0rp3n) [inline addons/sourcemod/translations/ttt.phrases.txt:240]: Could you correct the phrase as well here.

**Review by c0rp3n — CHANGES_REQUESTED** [review]
Thank you for doing this pass and also spotting the typo, could you just update the single phrase and then this can be merged.

File: addons/sourcemod/translations/ttt.phrases.txt (hunk 1 above)
Comment (lyczak) [reply addons/sourcemod/translations/ttt.phrases.txt:240]: I may be mistaken, but from what I gathered, it seemed like the strings prefacing bracket blocks are the internal phrases used to identify the actual block of localizable strings that gets shown in game. If this assumption is correct, then changing this block (1) would require additional refactoring inside of the actual code and (2) would not make any visible difference to a player. If I'm misinterpreting the way localizable strings are handled, please let me know so I can correct these remaining blocks.

Comment (c0rp3n) [reply addons/sourcemod/translations/ttt.phrases.txt:240]: It isnt shown to the user, just thought it will be best to rectify it in the code as well.

Comment (lyczak) [reply addons/sourcemod/translations/ttt.phrases.txt:240]: I agree, though those changes would be a bit more substantial and I'd want to make sure I don't miss anything. I'll take a look at it later if I get a chance.

**Comment by lyczak** [comment]
Internal strings have been updated now as well. There are a ton of configuration options that still need to be fixed as well as potentially inline comments too but perhaps those can be saved for another PR.

**Review by c0rp3n — APPROVED** [review]

**Comment by c0rp3n** [comment]
Yup we can look to fix that in a further change, thanks for doing this.

**Comment by Bara** [comment]
> Internal strings have been updated now as well. There are a ton of configuration options that still need to be fixed as well as potentially inline comments too but perhaps those can be saved for another PR.

🤣, but yeah most configs description was fastly written due fixes or some quick stuff.

**Pull request merged**